src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -573,6 +576,17 @@ def transform_request(self,
 
         return final_response
 
+    def _generate_manifest_uuid_name(self, filters: JSON) -> str:
+        """
+        Generate and return a UUID string generated using the latest git commit and filters
+        :param filters: The filters used in the manifest request
+        :return: String representation of a UUID
+        """
+        git_status = config.lambda_git_status
+        git_commit = git_status['commit']
+        filter_string = repr(sort_frozen(freeze(filters)))
+        return str(uuid.uuid5(uuid.NAMESPACE_DNS, git_commit + filter_string))
```

Comment:
A custom namespace seemed unnecessary given that we're generating a UUID to get a unique file name, and the git commit and filter string are the variable seed to the UUID generation. The namespace UUID just needs to be constant, and since the uuid module already has a constant available for use it seemed like a good fit.
I see how it's not that variable's intended use however and will re-add the custom namespace value